README.md

```diff
@@ -15,64 +37,6 @@ other characters. The goal is to correctly indent already
 well-structured templates, not to fix broken ones.
 
 
-### New! Multi-line HTML elements
-
-As of version 3, DjHTML indents multi-line HTML elements and
-multi-line attribute values like this:
-
-```jinja
-<blockquote cite="Guido Van Rossum"
-            style="font-style: italic;
-                   {% if dark_mode %}
-                       background: black;
-                   {% endif %}
-                  ">
-    Don't you hate code that's not properly indented?
-</blockquote>
-```
-
-
-### New! Multi-line CSS indentation
-
-Multi-line CSS values are now continued at the same indentation level:
-
-```jinja
-<style>
-    @font-face {
-        font-family: Helvetica;
-        src: {% for format, filename in licensed_fonts %}
-                 url('{% static filename %}') format('{{ format }}'),
-             {% endfor %}
-             url('Arial.woff2') format('woff2'),
-             url('Arial.woff') format('woff');
-    }
-</style>
-```
-
-
-### New! Improved JavaScript indentation
-
-Many new JavaScript indention rules have been added, such as the
-indentation of method chaining:
-
-```jinja
-<script>
-    window.fetch('/test.html')
-        .then((html) => {
-            document.body.innerHTML = html;
-            {% block extra_statements %}
-            {% endblock %}
-        });
-</script>
-```
-
-
-### New! Tabwidth guessing
-
-Without the `-t` / `--tabwidth` argument, DjHTML no longer defaults to
-a tabwidth of 4 but instead guesses the correct tabwidth.
-
-
```

Comment:
This was removed because it's hardly "new" anymore. I replaced it with a representative example of DjHTML's output at the top of this file.